rootfs/api/models.py

```diff
@@ -475,7 +475,7 @@ def _default_scale(self, user, release):
     def logs(self, log_lines=str(settings.LOG_LINES)):
         """Return aggregated log data for this application."""
         try:
-            url = "http://{}:{}/{}?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
+            url = "http://{}:{}/{}/?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
```

Comment:
This is the spot I thought I was commenting on before.  I checked and the extra slash isn't needed here.  But it also doesn't hurt, so I won't make a fuss. :)